<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1404</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TableOptionsDiv">
     <item>
      <widget class="QPushButton" name="getQueryButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; 
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="explainButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
color: rgb(255, 255, 255);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
       </property>
       <property name="text">
        <string>Explain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="transxButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; 
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Transx</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>1261</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="blocksScrollAreaWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>461</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 32, 32);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="BlocksDiv">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="MySGDBlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:22pt; color:#ececec;&quot;&gt;Tables&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>160</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="TablesVerticalLayout">
     <item>
      <widget class="QLabel" name="Table1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt; color:#ececec;&quot;&gt;table1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Table2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt; color:#ececec;&quot;&gt;table2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Table3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt; color:#ececec;&quot;&gt;table3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="CommandTextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>1081</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(33, 32, 32)</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="InfoHorizontalLayout">
     <item>
      <widget class="QLabel" name="TableOrderLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt; color:#ececec;&quot;&gt;Table Order&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="AmountRecordsLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt; color:#ececec;&quot;&gt;0 records&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SecondsDelayedLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 48, 30);
border : rgb(29, 185, 84);
border-radius: 10%; </string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt; color:#ececec;&quot;&gt;0 sec&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1404</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
